lib/jxl/encode.cc

```diff
@@ -191,6 +207,21 @@ JxlEncoderStatus JxlEncoderSetBasicInfo(JxlEncoder* enc,
       return JXL_ENC_ERROR;
       break;
   }
+  if (info->have_animation) {
+    enc->metadata.m.have_animation = true;
+    enc->metadata.m.animation.have_timecodes = false;
+    if (info->animation.tps_numerator > 0) {
+      enc->metadata.m.animation.tps_numerator = info->animation.tps_numerator;
+    } else {
+      enc->metadata.m.animation.tps_numerator = 100;
```

Comment:
this could return an error status, instead of choosing a default: require the user to set the tps_numerator correctly.